lib/lib-shared-models/src/main/java/org/sagebionetworks/repo/model/table/ViewTypeMask.java

```diff
@@ -1,10 +1,11 @@
 package org.sagebionetworks.repo.model.table;
 
 import org.sagebionetworks.repo.model.EntityType;
+import org.sagebionetworks.repo.model.EntityTypeUtils;
 
 public enum ViewTypeMask {
 
-	File(0x01, EntityType.file),
+	File(0x01, EntityTypeUtils.getFileTypes().toArray(new EntityType[0])),
```

Comment:
Think we are going to need a new type for views (can include only RecordSets or exclude them).